core/src/com/po/fuck/FUCK.java

```diff
@@ -24,14 +24,15 @@ public class FUCK extends ApplicationAdapter {
 
     SpriteBatch batch;
     private OrthographicCamera camera;
-
+    public static Managed<Coins> coinsManager;
```

Comment:
Manager is not a very good name, but I'm not sure what to use instead

Reply:
@MAKMED1337 ```CoinsCounter```?